Samples~/SampleGame/Assets/Tests/Editor/Unit/SampleTests.Unit.asmdef

```diff
@@ -12,10 +13,9 @@
     ],
     "excludePlatforms": [],
     "allowUnsafeCode": false,
-    "overrideReferences": true,
+    "overrideReferences": false,
```

Comment:
Not entirely sure how `overrideReferences` works, but does it need to be `true` to honor `precompiledReferences` defined below?

Reply:
Yeah this is needed, you can read up on asmdef here: https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html